docs/components/novatalkscore.md

```diff
@@ -47,14 +63,14 @@
 		-  Output embedded text from the menu + the ability to attach a file
 
 
-[Menu (IVR) TK](https://drive.google.com/drive/folders/1yrgBr7YQ8wh_YuhyDZn6E-RhJZgkqzkv)
+[Menu (IVR) specification](https://drive.google.com/drive/folders/1yrgBr7YQ8wh_YuhyDZn6E-RhJZgkqzkv)
 
 
-[ChatBot TK](https://drive.google.com/drive/folders/1wl5w_mNff59Cdrz0BJfL9iI3zrFnDCjw)
+[ChatBot specification](https://drive.google.com/drive/folders/1wl5w_mNff59Cdrz0BJfL9iI3zrFnDCjw)
```

Comment:
Link is protected with authorization check, recheck if this is a valid folder on disk